package.json

```diff
@@ -1,7 +1,7 @@
 
 {
   "name": "react-native-android-fragment",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I'm thinking it might make sense to start creating release tags again - and this could be release `1.0.1`?